### ISSUE TITLE
Handle link feature can't open new Reddit app link (reddit.app.link)

### ISSUE THREAD

**Issue by Dangelman** [Docile-Alligator/Infinity-For-Reddit#1023]
### Checklist

- [x] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

5.2.1

### Where did you get the app from

Google Play

### Android version

11

### First occurred

Couple days?

### Steps to reproduce

Reddit introduced a new experimental link for the app, reddit.app.link, when you try sharing a post from the reddit app.

Infinity can't open it with the link handler feature (#754).

![image](https://user-images.githubusercontent.com/26676154/185789000-74d1dfb3-288f-46e8-896c-dbfcde93ff47.png)

**Comment by Wladefant** [comment]
It is already supported

**Comment by Dangelman** [comment]
Oh, sorry, then it's just the "handle link" feature that doesn't support the reddit.app.link
Let me rephrase the issue @Wladefant

**Comment by Wladefant** [comment]
Can not reproduce it, the link is handled correctly
